swagger.json

```diff
@@ -1120,7 +1120,7 @@
                             "type": "string"
                         },
                         "example": [
-                            "did:cheqd:testnet:ca9ff47c-0286-4614-a4be-8ffa83911e09#key-0"
+                            "did:cheqd:testnet:7bf81a20-633c-4cc7-bc4a-5a45801005e0#key-0"
```

Comment:
It seems that should be `did:cheqd:testnet:7bf81a20-633c-4cc7-bc4a-5a45801005e0#key-1` instead of `did:cheqd:testnet:7bf81a20-633c-4cc7-bc4a-5a45801005e0#key-0`.